main.ts

```diff
@@ -423,6 +403,51 @@ async (req: Request) => {
 export const supported_nips = [1, 2, 11];
 export const software = "https://github.com/BlowaterNostr/relayed";
 
+const graphql_login_handler =
+    (args: { relayInformationStore: RelayInformationStore }) => async (req: Request) => {
+        const body = await req.json();
+        if (!body) {
+            return new Response(`{"errors":"request body is null"}`, { status: 400 });
+        }
+        const error = await verifyToken(body, args.relayInformationStore);
+        if (error instanceof Error) {
+            return new Response(JSON.stringify(error.message), { status: 400 });
+        } else {
+            const auth = btoa(JSON.stringify(body));
+            const headers = new Headers();
+            const cookie: Cookie = {
+                name: "token",
+                value: auth,
+                path: "/",
+                secure: true,
+                httpOnly: true,
+                sameSite: "Strict",
+            };
+            setCookie(headers, cookie);
+            const resp = new Response("", { status: 200, headers });
+            return resp;
+        }
+    };
+
+const members_handler = async (args: { get_space_members: func_GetSpaceMembers }) => {
+    const members = await args.get_space_members();
+    if (members instanceof Error) {
+        console.log(members);
+        return new Response("", { status: 500 });
+    }
+    const body = {
+        data: members.map((event) => event.member),
```

Comment:
why wrap it with `data`?

Reply:
It is also possible to return the array directly, wrap it with `data` is a personal habit